docs/pages/release_notes.md

```diff
@@ -14,13 +14,35 @@ This is a {{ site.pmd.release_type }} release.
 
 ### New and noteworthy
 
+#### CLI improvements
+
+The PMD CLI now allows repeating the `--dir` (`-d`) and `--rulesets` (`-R`) options,
+ as well as providing several space-separated arguments to either of them. For instance:
+```shell
+pmd -d src/main/java src/test/java -R rset1.xml -R rset2.xml
+```
+This also allows globs to be used on the CLI if your shell supports shell expansion.
+For instance, the above can be written
+```shell
+pmd -d src/*/java -R rset*.xml
+```
+Please use theses new forms instead of using comma-separated list as argument to these options.
```

Comment:
English
```suggestion
Please use theses new forms instead of using comma-separated lists as argument to these options.
```

Reply:
I guess, I missed that...

Reply:
Added with 01c48e1ca7686fec520a687d30b8bb2e74b09ce6